apps/remix-ide/src/app/tabs/locales/en/udapp.json

```diff
@@ -108,9 +108,9 @@
   "udapp.tooltipText13": "Deployed {date}",
 
   "udapp._comment_universalDappUI.tsx": "libs/remix-ui/run-tab/src/lib/components/universalDappUI.tsx",
-  "udapp.tooltipTextRemove": "Remove from the list",
+  "udapp.tooltipTextRemove": "Remove permanently from the list",
   "udapp.tooltipTextEdit": "Create a DApp using this contract in the main panel",
-  "udapp.tooltipTextPin": "Pin contract",
+  "udapp.tooltipTextPin": "Pin contract for select workspace and network to persist after reload",
```

Comment:
A pinned contract will be persisted after a browser refresh as long as the same Workspace and network are selected.

Reply:
this doesn't define the action of button and is more like a sentence. May be we can rephrase this